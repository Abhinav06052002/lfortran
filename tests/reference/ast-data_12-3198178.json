{
    "basename": "ast-data_12-3198178",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/data_12.f90",
    "infile_hash": "07a085ad3b761c6644390f860922eac4de21df14fc975f684a702904",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-data_12-3198178.stdout",
    "stdout_hash": "ae622aadb41d603fdc7952440912daff6f79a1c91f96e7e7565f4e65",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}