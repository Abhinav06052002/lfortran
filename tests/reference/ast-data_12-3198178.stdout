(TranslationUnit
    [(Module
        m
        (TriviaNode
            []
            [(EndOfLine)
            (EndOfLine)]
        )
        []
        [(ImplicitNone
            []
            ()
        )]
        [(Declaration
            (AttrType
                TypeInteger
                []
                ()
                ()
                None
            )
            []
            [(x
            []
            []
            ()
            ()
            None
            ())]
            ()
        )]
        [(DataStmt
            0
            [(DataStmtSet
                [x]
                [1]
            )]
            ()
        )]
        []
    )
    (Program
        main
        ()
        []
        [(ImplicitNone
            []
            ()
        )]
        []
        []
        []
    )]
)
